code/index.html

```diff
@@ -3,20 +3,127 @@
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css"
+      integrity="sha512-ZYg+hQvLlM0W9C3P6vTpes5LmJ66G2EsroxDNzwD6qiulckldb29eD9I672HL2X+LH5nRhY1sQqZLJGB+aUUPg=="
+      crossorigin="anonymous"
+    />
+
+    <!--Allow us to use icons-->
+
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Newsreader&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Hina+Mincho&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <nav>
+        <a>notion</a>
+        <a>stack<b>overflow</b></a>
+        <a>slack</a>
+      </nav>
+
+      <div>
+        <img src="./logo.svg" class="logo" />
+      </div>
+
+      <div>
+        <h1>TECHNIGO'S NEWS</h1>
+      </div>
     </header>
 
     <section class="big-news">
       <!-- The big news content -->
+      <div class="cat-foto-wrapper">
+        <img src="./Pictures/Cat-picture.jpg" alt="British short hair cat" />
+      </div>
+
+      <div class="big-news-text">
+        <h2>WE ARE SENDING TO ALL OF YOU A CAT!</h2>
+        <p>
+          Recently studies have shown that cat owners do have better
+          psychological health than people without pets. They help you sleeping
+          better, feeling less stressed and anxious, and even their purring
+          helps to heal tendons and muscles!<br /><br />
```

Comment:
I think its generally recommended to not use <br> tag because it is not semantic. Don't you think it would be batter to divide text in two paras using two <P> tags?